front/components/assistant/conversation/AgentMessage.tsx

```diff
@@ -94,19 +95,20 @@ export const FeedbackSelectorPopoverContent = ({
 
   return (
     agentLastAuthor && (
-      <div className="itemcenter mt-4 flex gap-2">
-        {agentLastAuthor?.image && (
-          <img
-            src={agentLastAuthor?.image}
-            alt={agentLastAuthor?.firstName}
-            className="h-8 w-8 rounded-full"
-          />
-        )}
-        <Page.P variant="secondary">
-          Your feedback will be sent to:
-          <br />
-          {agentLastAuthor?.firstName} {agentLastAuthor?.lastName}
-        </Page.P>
+      <div className="mb-4 mt-2 flex flex-col gap-2">
+        <Page.P variant="secondary">Your feedback goes to:</Page.P>
```

Comment:
I think we discussed this but this is really a lie. The feedback does not go to the last author in particular and it is actually visible by the entire workspace.

Can we tweak the wording here? :pray: 

cc @pinotalexandre

Reply:
Happy to change it, we could go with "Your feedback is publicly available. The last assistant author is:", feel free to suggest anything

Reply:
This is much closer to the truth :+1:

Reply:
publicly is a bit of an exagerration. Your feedback is available to the workspace maybe?